README.md

```diff
@@ -1,68 +1,51 @@
 # OLLA
 
-OLLA (Optimizing the Lifetime and Location of Arrays) makes it possible to train larger deep neural networks on existing hardware. OLLA optimizes the order in which the neural network operators are executed to minimize peak memory usage. Furthermore OLLA eliminates memory fragmentation to ensure that no memory is wasted.
+*OLLA* (Optimizing the Lifetime and Location of Arrays) enables training larger deep neural networks on existing hardware. It accomplishes this with a few techniques:
+- *Operator order optimization* — reodering tensor operators to reduce peak memory usage
+- *Fragmentation reduction* — dynamic memory profiling and scheduling to better-utilize memory.
 
-## Approach
-
-Our approach is described in detail on the [OLLA arXiv paper](https://arxiv.org/abs/2210.12924)
+Our approach is described in detail on the [OLLA arXiv paper](https://arxiv.org/abs/2210.12924). See [citing](#citation) below to attribute the work.
 
 ## Getting Started
-
-Install
+The following steps are required to run the optimizer:
 ```
-conda create --name olla python=3.9
-conda activate olla
-conda install pytorch torchvision torchaudio torchtext -c pytorch
-pip install pandas intervaltree networkx graphviz
-conda config --add channels http://conda.anaconda.org/gurobi
-conda install gurobi
-pip install pydot
+pip install networkx graphviz intervaltree pandas pydot
```

Comment:
also in the future we should add versions of those libraries that are currently working, or put them in requirements.txt, to avoid breaking changes from those libraries.